test/e2e/admission.go

```diff
@@ -17,14 +17,14 @@ limitations under the License.
 package e2e
 
 import (
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
```

Comment:
let's keep this to make coding simple.

Reply:
Ok. I have updated it and raised a PR #234 .